clientv3/integration/leasing_test.go

```diff
@@ -619,16 +619,29 @@ func TestLeasingTxnOwnerGet(t *testing.T) {
 	clus := integration.NewClusterV3(t, &integration.ClusterConfig{Size: 1})
 	defer clus.Terminate(t)
 
-	lkv, closeLKV, err := leasing.NewKV(clus.Client(0), "pfx/")
+	client := clus.Client(0)
+
+	lkv, closeLKV, err := leasing.NewKV(client, "pfx/")
 	testutil.AssertNil(t, err)
 	defer closeLKV()
 
+	defer func() {
+		// In '--tags cluster_proxy' mode the client need to be closed before
```

Comment:
Maybe move CloseLKV() inside this block for better clarity?

Reply:
Done.